### DISCUSSION TITLE
安装 MySQL - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#918] [Announcements]
# 安装 MySQL - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/mysql/quickstart/install/

**Comment by liyuerich** [comment]
输入合适的名称，选择命名空间和版本后，点击确定。也可以在底部配置 YAML 来安装。
能给个sample，怎么装离线（私有仓库）的mysql-operator吗？
